GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -333,12 +366,115 @@ public void Clear()
             Invalidate(invalidateChildren: true);
         }
 
+        public void EndSelectionAtLoad()
```

Comment:
It would be good to have docs for this method explaining what it does and why it is necessary.
Ditto for other public API.

Reply:
Maybe call this `ClearPreviousSelectionCache`?

Reply:
`ClearToBeSelected()`